.env.example

```diff
@@ -1,8 +1,12 @@
 APP_PORT=8080
 APP_HOST=localhost
+APP_ENV=development
 
 PG_HOST=127.0.0.1
 PG_PORT=5432
 PG_NAME=postgres
 PG_USER=root
-PG_PASSWORD=
\ No newline at end of file
+PG_PASSWORD=
+
+BASIC_AUTH_USERNAME=username
```

Comment:
What is purpose of adding basic auth username and password?

Reply:
ini untuk case register admin nya mas, jadi katanya kan kemaren harus pake token
jadi alternatif nya di kasih basic auth aja klo api register admin